README.md

```diff
@@ -1,52 +1,88 @@
 # Kik Bot API #
-Use this library to develop bots for [Kik Messenger](https://www.kik.com) that are essentially automated humans.
+The Unofficial Kik Bot API is a Python library developed to automate interactions on [Kik Messenger](https://www.kik.com) that are essentially automated humans.
```

Comment:
The interactions are not automated humans, that bots are. Maybe we can remove the "that are essentially automated humans" part.